examples/collector-exporter-node/README.md

```diff
@@ -0,0 +1,39 @@
+# Overview
+
+This example shows how to use [@opentelemetry/exporter-collector](https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-exporter-collector) to instrument a simple Node.js application - e.g. a batch job.
```

Comment:
What do you mean by batch job here?

Reply:
this has been copied from `basic-tracer-node` example